bokeh/server/views/autoload_js_handler.py

```diff
@@ -8,6 +8,7 @@
 
 from six.moves.urllib.parse import urlparse
 from tornado import gen
+from tornado.web import MissingArgumentError
```

Comment:
It will have to be removed in this PR, unused imports trigger hard CI test failures:
```
_________________________________ test_flake8 __________________________________
    @pytest.mark.quality
    def test_flake8():
        chdir(TOP_PATH)
    
        proc = Popen(["flake8"], stdout=PIPE, stderr=PIPE)
        out, err = proc.communicate()
    
>       assert proc.returncode == 0, "Flake8 issues:\n%s" % out.decode("utf-8")
E       AssertionError: Flake8 issues:
E         ./bokeh/server/views/autoload_js_handler.py:11:1: F401 'tornado.web.MissingArgumentError' imported but unused
E         ./build/lib/bokeh/server/views/autoload_js_handler.py:11:1: F401 'tornado.web.MissingArgumentError' imported but unused
E         
E       assert 1 == 0
E        +  where 1 = <subprocess.Popen object at 0x7f0c7d504710>.returncode
tests/test_flake8.py:16: AssertionError
```
You should just be able to make the change in a new commit and push your branch, and it will update this PR automatically.